platform/accessibility/README.md

```diff
@@ -1,24 +1,77 @@
+
+## How to contact us
+VSP Accessibility contact: Trevor Pierce, `@Trevor Pierce` in Slack, `1Copenut` in Github
+Slack channels: #vetsgov-accessibility
+Github labels: `508/Accessibility`
 
-### Communication channels
-- Slack channels: #vetsgov-accessibility
-- Github labels: `508/Accessibility`
+## What we do
+VSP Accessibility ensures that accessibility and inclusive design are an integral part of product design and development on VA.gov.
 
-### VSP POCs
-- Trevor Pierce, @trevorpierce in Slack, @1Copenut in Github
+* Accessibility is an outcome, generally linked to compliance, ensuring solutions are usable by everyone.
+* Inclusive design is a human-centered process considering user diversity, by including and learning from a range of perspectives.
 
-### Table of contents
-The following list is a set of resources to help you do your work and answer questions you might have. Some of the links may take you to other sites and resources outside this repo. If you need a hand with anything, don't hesitate to ping the #[slack-channel] Slack channel or your VSP POC directly.
+## Working with us
+### VSP Collaboration Cycle
+The [VSP Collaboration Cycle](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/working-with-vsp/vsp-collaboration-cycle) defines the process for VSP support of VFS teams to ensure that products and services on [VA.gov](https://www.va.gov/) are human-centered, reliable, and continually improving
 
+Accessibility guidance is included in the following phases of the VSP Collaboration Cycle:
 
-* [Accessibility Best Practices](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-accessibility-best-practices.md)
-* [508 Checklist](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-checklist.md)
-* [WCAG Checklist](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/WCAG-Checklist.md)
+* **Project Kickoff:** Incorporating accessibility into the early phases of a project minimizes the need for rework later. Relevant resources: [508 checklist](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-checklist.md), [508, accessibility, and inclusive design best practices](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-accessibility-best-practices.md).
+* **Define:** We help in early project discussions to make sure inclusive design is woven into your work. Main areas to consider at this phase include:
+   * challenges with user flows
+   * color contrast
+   * color blindness
+   * accessible interactions
+   * design for various device sizes
+   * cognitive impact
+* **Discover:** We provide more specific feedback at this phase, including:
+   * relevant design system components and patterns
+   * feedback on recruiting diverse research participants
```

Comment:
As mentioned in the Define stage, considerations for planning to include diverse research would be helpful earlier, as recruiting people with disabilities can take quite a bit of time, as can planning research.